front-end/vue.config.js

```diff
@@ -1,7 +1,7 @@
 const webpack = require('webpack')
 	
 module.exports = {
-  publicPath: './',
+  publicPath: '/',
```

Comment:
This **might** break prod if I'm reading it right, what was the reason for this change?

Reply:
Nice catch!

I see what you mean. This is required for Dev to work w/ history mode router. If you go to a nested page like `/layer/:uuid` then it tries to pull assets from `/layer/public/etc` and breaks.

But in prod we have everything living behind a proxied route, which is a special case. I can make sure to resolve this by having a production-specific configuration.